lib/tilex_web/templates/post/show.html.eex

```diff
@@ -3,9 +3,3 @@
   post: @post,
   twitter_shareable: assigns[:twitter_shareable]
 %>
-
-<%= if TilexWeb.PostView.more_info?(@post.channel) do %>
-  <div class="more-info">
-    <%= render TilexWeb.SharedView, "_#{@post.channel.name}.html" %>
-  </div>
-<% end %>
```

Comment:
This was loading the files in `lib/tilex_web/templates/shared` based on what the tag is on the post. I also removed all of those shared files.